dipy/reconst/tests/test_eudx_dg.py

```diff
@@ -1,10 +1,7 @@
-import warnings
-
 import numpy as np
 import numpy.testing as npt
 
 from dipy.direction.peaks import default_sphere, peaks_from_model
-import dipy.reconst.eudx_direction_getter
```

Comment:
Weird. The name of this file and the name of the test make no sense.

Reply:
Thanks for the review @arokem. I do agree that the file name and contents seem not to match closely the `dipy.reconst.eudx_direction_getter.pyx`, which is the one that I guess should be targeted. Not sure about the intent of the current content: maybe the test's contents should be assimilated to the `direction.tests.test_peaks` or maybe was initially targeted to exercise `direction.closest_peak_direction_getter.pyx` (which does not have a dedicated test). To me, the pointed file should be reworked to exercise the EulerDX direction getter. Something similar happens to the `dipy.reconst.test_peak_finding.py` test (should maybe renamed to `test_recspeed.py`). Best would be to address them in a separate PR or to keep track of them through an issue.

Reply:
Yes, let's do that in a separate PR. I will go ahead and merge this PR. Thank you @jhlegarreta